### PR TITLE
Add release note for v2.0

### PR DESCRIPTION
**Pull request by kvartet**
Shall we update the section headers because of the strange sidebar in readthedocs?

image

### CODE REVIEW

File: docs/en_US/Release.rst [hunk 1]
```diff
@@ -5,6 +5,99 @@
 ChangeLog
 =========
 
+Release 2.0 - 11/1/2021
```
Comment (scarlett2018) [inline docs/en_US/Release.rst:8]: to be updated

File: docs/en_US/Release.rst [hunk 2]
```diff
@@ -5,6 +5,99 @@
...
+
+Major updates
+-------------
+
+Neural architecture search
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Support an improved NAS framework: Retiarii (experimental)
+
+  * Feature roadmap (TODO)
```
Comment (scarlett2018) [inline docs/en_US/Release.rst:19]: @QuanluZhang to add the Retiarii future work issue.

File: docs/en_US/Release.rst [hunk 3]
```diff
@@ -5,6 +5,99 @@
...
+  * Related issues and pull requests (label: retiarii-2.0)
```
Comment (scarlett2018) [inline docs/en_US/Release.rst:20]: change the (label: retiarii-2.0) to an url for easy access, something like below:

[Related issues and PR](https://github.com/microsoft/nni/issues?q=label%3Aretiarii-2.0+)

Comment (scarlett2018) [reply docs/en_US/Release.rst:20]: Same comments to all the issues, add like to them. i.e. #3221

**Review by scarlett2018 — CHANGES_REQUESTED** [review]

File: docs/en_US/Release.rst (hunk 1 above)
Comment (kvartet) [reply docs/en_US/Release.rst]: updated it.

File: docs/en_US/Release.rst (hunk 3 above)
Comment (kvartet) [reply docs/en_US/Release.rst]: updated and added.

File: docs/en_US/Release.rst [hunk 4]
```diff
@@ -5,6 +5,97 @@
...
+-------------
+
+Neural architecture search
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Support an improved NAS framework: Retiarii (experimental)
+
+  * Feature roadmap (TODO)
+  * `Related issues and pull requests <https://github.com/microsoft/nni/issues?q=label%3Aretiarii-v2.0>`__
+  * Documentations (#3221 #3282 #3287)
```
Comment (liuzhe-lz) [inline docs/en_US/Release.rst:21]: I think "documentation" already means "document-s", so "s" is not needed.

File: docs/en_US/Release.rst [hunk 5]
```diff
@@ -5,6 +5,97 @@
...
+
+* Support fpgm in pruning schedule (#3110)
+* Update graphutils to support torch v1.7 (#3076)
+* Update flops counter (#3048 #3265)
+
+
+WebUI & nnictl 
+^^^^^^^^^^^^^^
+
+* Add all experiments management page (#3081 #3127)
```
Comment (liuzhe-lz) [inline docs/en_US/Release.rst:44]: I think "add page to manage all experiments" is easier to understand.

File: docs/en_US/Release.rst [hunk 6]
```diff
@@ -5,6 +5,97 @@
...
+* Update the layout of overview page (#3046 #3123)
+* Update right nav bar, log position and expanded icons for table (#3069 #3103)
+* Unify nnictl port type (#3142)
```
Comment (liuzhe-lz) [inline docs/en_US/Release.rst:47]: This one don't need to be mentioned in release note.

File: docs/en_US/Release.rst [hunk 7]
```diff
@@ -5,6 +5,97 @@
...
+
+
+Others
+^^^^^^
+
+* Support launching an experiment from Python code (#3111 #3210 #3263)
+* Reorganize Python package, package hierarchy, source code directory hierarchy (#2962 #2987 #3037)
```
Comment (liuzhe-lz) [inline docs/en_US/Release.rst:54]: "Reorganize source code directory hierarchy" is enough.

File: docs/en_US/Release.rst [hunk 8]
```diff
@@ -5,6 +5,97 @@
...
+* Refactor builtin/customized tuner installation (#3134)
+* Add experiment configuration V2 (#3138)
+* Change SIGKILL to SIGTERM in local mode to cancel a trial job (#3173)
+* Refector hyperband (#3040)
+* Refactor HPO tuner code hierarchy (#3187)
+* Unify the names of trail job id (#3053)
+
+
+UT & IT
```
Comment (liuzhe-lz) [inline docs/en_US/Release.rst:63]: This section is not needed.

File: docs/en_US/Release.rst [hunk 9]
```diff
@@ -5,6 +5,97 @@
...
+  * Documentations (#3221 #3282 #3287)
+
+* Support a new NAS algorithm: Cream (#2705)
+* Add one NAS NLP benchmark (#3140)
+
+Training service
+^^^^^^^^^^^^^^^^
+
+* Support hybrid training service (#3097 #3251)
+* Support a new training service implementation for Kubernetes (#3022, thanks external contributors Petuum @pw2393)
```
Comment (liuzhe-lz) [inline docs/en_US/Release.rst:30]: Better mention the training service's name.

File: docs/en_US/Release.rst [hunk 10]
```diff
@@ -5,6 +5,97 @@
...
+* Support custom Kubernetes namespace in AdaptDL mode (#3176)
```
Comment (liuzhe-lz) [inline docs/en_US/Release.rst:31]: Not needed. It's part of previous feature.

File: docs/en_US/Release.rst [hunk 11]
```diff
@@ -5,6 +5,97 @@
...
+Others
+^^^^^^
+
+* Support launching an experiment from Python code (#3111 #3210 #3263)
+* Reorganize Python package, package hierarchy, source code directory hierarchy (#2962 #2987 #3037)
+* Refactor builtin/customized tuner installation (#3134)
+* Add experiment configuration V2 (#3138)
+* Change SIGKILL to SIGTERM in local mode to cancel a trial job (#3173)
+* Refector hyperband (#3040)
+* Refactor HPO tuner code hierarchy (#3187)
```
Comment (liuzhe-lz) [inline docs/en_US/Release.rst:59]: Not needed. Part of Line 54's feature.

File: docs/en_US/Release.rst [hunk 12]
```diff
@@ -5,6 +5,97 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Support an improved NAS framework: Retiarii (experimental)
+
+  * Feature roadmap (TODO)
+  * `Related issues and pull requests <https://github.com/microsoft/nni/issues?q=label%3Aretiarii-v2.0>`__
+  * Documentations (#3221 #3282 #3287)
+
+* Support a new NAS algorithm: Cream (#2705)
+* Add one NAS NLP benchmark (#3140)
```
Comment (ultmaster) [inline docs/en_US/Release.rst:24]: NAS NLP benchmark is enough.

File: docs/en_US/Release.rst [hunk 13]
```diff
@@ -5,6 +5,97 @@
...
+^^^^^^
+
+* Support launching an experiment from Python code (#3111 #3210 #3263)
+* Reorganize Python package, package hierarchy, source code directory hierarchy (#2962 #2987 #3037)
+* Refactor builtin/customized tuner installation (#3134)
+* Add experiment configuration V2 (#3138)
+* Change SIGKILL to SIGTERM in local mode to cancel a trial job (#3173)
+* Refector hyperband (#3040)
+* Refactor HPO tuner code hierarchy (#3187)
+* Unify the names of trail job id (#3053)
```
Comment (liuzhe-lz) [inline docs/en_US/Release.rst:60]: Not needed. Internal change.

File: docs/en_US/Release.rst [hunk 14]
```diff
@@ -5,6 +5,97 @@
...
+
+Documentation
+-------------
+
+* Port markdown docs to reStructuredText docs and introduce ``githublink`` (#3107)
+* List related research and publications (#3150)
+* Add quantized model export description (#3192)
+* Add documentation for launching from python code (#3217)
+* Remove paiYarn doc and add ``reuse`` config in remote doc (#3253)
+* Change quick start example to PyTorch and update installation from source doc (#3266)
```
Comment (ultmaster) [inline docs/en_US/Release.rst:78]: This is part of "update code hierarchy"

File: docs/en_US/Release.rst [hunk 15]
```diff
@@ -5,6 +5,97 @@
...
+* Update unit test and integration test for code hierarchy refactor (#3037 #3088)
+* Update integration test for AdaptDL (#3153)
+
+Documentation
+-------------
+
+* Port markdown docs to reStructuredText docs and introduce ``githublink`` (#3107)
+* List related research and publications (#3150)
+* Add quantized model export description (#3192)
+* Add documentation for launching from python code (#3217)
```
Comment (ultmaster) [inline docs/en_US/Release.rst:76]: This is part of "launch from python code"

File: docs/en_US/Release.rst [hunk 16]
```diff
@@ -5,6 +5,97 @@
...
+* Remove paiYarn doc and add ``reuse`` config in remote doc (#3253)
+* Change quick start example to PyTorch and update installation from source doc (#3266)
+* Update EfficientNet doc to clarify repo versions (#3158, thanks external contributor @ahundt)
+
+Bug fixes
+---------
+
+* Fix exp-duration pause timing under no_more_trial status (#3043)
+* Fix bug in nas SPOS trainer, apply_fixed_architecture (#3051, thanks external contributor @HeekangPark)
+* Fix ``_compute_hessian`` bug in Pytorch NAS Darts (#3058, thanks external contributor @hroken)
```
Comment (ultmaster) [inline docs/en_US/Release.rst:86]: DARTS

**Review by liuzhe-lz — APPROVED** [review]

**Review by ultmaster — APPROVED** [review]

**Review by QuanluZhang — APPROVED** [review]

File: docs/en_US/Release.rst [hunk 17]
```diff
@@ -5,6 +5,84 @@
...
+* Fix ``_compute_hessian`` bug in NAS DARTS (PyTorch version) (#3058, thanks external contributor @hroken)
+* Fix bug of conv1d in the cdarts utils (#3073, thanks external contributor @athaker)
+* Fix the handling of unknown trials when resuming an experiment (#3096)
+* Fix bug of kill command under Windows (#3106)
+* Fix lazy logging (#3108, thanks external contributor @HarshCasper)
+* Fix checkpoint load and save issue in QAT quantizer (#3124, thanks external contributor @eedalong)
+* Fix quant grad function calculation error (#3160, thanks external contributor @eedalong)
+* Fix device assignment bug in quantization algorithm (#3212, thanks external contributor @eedalong)
+* Fix bug in ModelSpeedup and enhance UT for it (#3279)
+* and others
```
Comment (scarlett2018) [inline docs/en_US/Release.rst:83]: link to others?

**Review by scarlett2018 — CHANGES_REQUESTED** [review]

File: docs/en_US/Release.rst (hunk 17 above)
Comment (kvartet) [reply docs/en_US/Release.rst]: added.

**Review by scarlett2018 — APPROVED** [review]

**Pull request merged**